front_end/panels/ai_chat/core/ToolNameMapping.test.ts

```
@@ -0,0 +1,84 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { createAgentNode, createToolExecutorNode } from './AgentNodes.js';
```

ESLint

[error] Incorrect same-namespace import: "./AgentNodes.js". Use "import * as Core from './core.js';" instead.

(rulesdir/es-modules-import)